Setup.bat

```diff
@@ -57,7 +57,10 @@ call :MarkStartOfBlock "Setup variables"
     set SCHEMA_COPY_DIR=%~dp0..\..\..\spatial\schema\unreal\gdk
     set SCHEMA_STD_COPY_DIR=%~dp0..\..\..\spatial\build\dependencies\schema\standard_library
     set SPATIAL_DIR=%~dp0..\..\..\spatial
-
+	set DOMAIN_ENVIRONMENT_VAR=
+	FOR %%A IN (%*) DO (
+		IF "%%A"=="/China" set DOMAIN_ENVIRONMENT_VAR=--domain spatialoschina.com --environment cn-production

```

Comment:
I think Mike is also adding a flag to `Setup.bat`. I think it would be good to sync on the flag format as I believe he's taking a `--` approach. Additionally we should probably sync with our other tooling with uses `--`

Reply:
Let's also make sure we get a docs update prepared for this. Do we have a docs ticket in progress?

Reply:
Changed to --, ill ping Mike to make sure 

There's no docs ticket, how can I create one?